### PR TITLE
Add example for `duplicate-bases` message documentation

### PR DESCRIPTION
**Pull request by gunungpw**
Co-authored-by: Vladyslav Krylasov <vladyslav.krylasov@gmail.com>

- [x] Write a good description on what the PR does.
- [x] If you used multiple emails or multiple names when contributing, add your mails
   and preferred name in ``script/.contributors_aliases.json``

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
|    | :bug: Bug fix          |
|    | :sparkles: New feature |
|    | :hammer: Refactoring   |
| ✓   | :scroll: Docs          |

## Description

add code example for `duplicate-bases`
add python traceback error message
add Issue Tracker link

Ref #5953

### CODE REVIEW

File: doc/data/messages/d/duplicate-bases/details.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.. code:: shell-session
```
Comment (DanielNoord) [inline doc/data/messages/d/duplicate-bases/details.rst:1]: I didn't know this raised a `TypeError`. Could you add this to the description of `duplicate-bases`  in its declaration in https://github.com/PyCQA/pylint/blob/main/pylint/checkers/classes/class_checker.py?

We can then remove this file.

File: doc/data/messages/d/duplicate-bases/related.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+`Issue Tracker <https://github.com/PyCQA/pylint/issues?q=is%3Aissue+%22inconsistent-mro%22+OR+%22E0240%22>`_
```
Comment (DanielNoord) [inline doc/data/messages/d/duplicate-bases/related.rst:1]: See my comments on the other PR you openend :)

**Review by DanielNoord — CHANGES_REQUESTED** [review]
Thanks again @gunungpw

File: doc/data/messages/d/duplicate-bases/related.rst (hunk 2 above)
Comment (gunungpw) [reply doc/data/messages/d/duplicate-bases/related.rst:1]: Thank you ;)

File: doc/data/messages/d/duplicate-bases/details.rst (hunk 1 above)
Comment (gunungpw) [reply doc/data/messages/d/duplicate-bases/details.rst:1]: Done ✔️

File: pylint/checkers/classes/class_checker.py [hunk 3]
```diff
@@ -613,7 +613,7 @@ def _has_same_layout_slots(slots, assigned_value):
     "E0241": (
         "Duplicate bases for class %r",
         "duplicate-bases",
-        "Used when a class has duplicate bases.",
+        "Duplicate use of base class in derived class are TypeError.",
```
Comment (DanielNoord) [inline pylint/checkers/classes/class_checker.py:616]:
```suggestion
        "Duplicate use of base classes in derived classes raise TypeErrors.",
```

Comment (gunungpw) [reply pylint/checkers/classes/class_checker.py]: Thank you, English is not my first language 🙏

**Comment by gunungpw** [comment]
Any advice why `Checks / documentation (pull_request) ` fail? I am new to programing and Github 😄

**Comment by DanielNoord** [comment]
@gunungpw I'm scared it might be due to #6060.

If you want to fix it you would likely need to look for the `add_message` calls that have `"duplicate-bases"` as their first argument. Then figure it by which `visit_...` or `leave_...` method (I guess `visit_classdef`) they get called and check if `duplicate-bases` is in their `check_messages` decorator. I know that is quite a lot so if you have trouble figuring this out let me know and I can take a look myself!

File: doc/data/messages/d/duplicate-bases/bad.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+class Animal:
+    pass
+
+
+class Cat(Animal, Animal):
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/d/duplicate-bases/bad.py:5]:
```suggestion
class Cat(Animal, Animal):  # [duplicate-bases]
```
At least I suppose it's what we expect.

Comment (gunungpw) [reply doc/data/messages/d/duplicate-bases/bad.py]: Thank you, I forget it :)

Comment (DanielNoord) [reply doc/data/messages/d/duplicate-bases/bad.py]: How could I have missed that 😅

**Comment by gunungpw** [comment]
> @gunungpw I'm scared it might be due to #6060.
> 
> If you want to fix it you would likely need to look for the `add_message` calls that have `"duplicate-bases"` as their first argument. Then figure it by which `visit_...` or `leave_...` method (I guess `visit_classdef`) they get called and check if `duplicate-bases` is in their `check_messages` decorator. I know that is quite a lot so if you have trouble figuring this out let me know and I can take a look myself!

I learned a lot today, the issue may be a bit too complex for me.
Hope you have a nice day

**Comment by DanielNoord** [comment]
> > @gunungpw I'm scared it might be due to #6060.
> > If you want to fix it you would likely need to look for the `add_message` calls that have `"duplicate-bases"` as their first argument. Then figure it by which `visit_...` or `leave_...` method (I guess `visit_classdef`) they get called and check if `duplicate-bases` is in their `check_messages` decorator. I know that is quite a lot so if you have trouble figuring this out let me know and I can take a look myself!
> 
> I learned a lot today, the issue may be a bit too complex for me. Hope you have a nice day

@Pierre-Sassoulas Actually already fixed it by adding the comment. 😄

**Review by Pierre-Sassoulas — APPROVED** [review]
Thank you for the contribution @gunungpw , and congratulation on becoming a contributor !

**Pull request merged**